src/main/java/ch/hearc/todont/controllers/CreateToDontController.java

```diff
@@ -40,18 +44,22 @@ public String get(Model model) {
     /**
      * Post to add new to-dont.
      * 
-     * @param user      User adding the to-dont
+     * @param userDetail User adding the to-dont
      * @param newToDont Returned value from form
      * @return The page template name
      */
     @PostMapping("/create")
     public String createToDont(@RequestParam(name = "usernames[]") String[] usernames, 
         @RequestParam(name = "moderatorBoolean[]") String[] moderators,
-        @AuthenticationPrincipal User user,
+        @AuthenticationPrincipal org.springframework.security.core.userdetails.User userDetail,
```

Comment:
Could you please copy this to all the mappings that use the User's info ? Along with the line 57